### PR TITLE
Solomon places

### PR DESCRIPTION
**Pull request by sologebre**

### CODE REVIEW

File: new/LOC7318Abazaz.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <revisionDesc>
+            <change who="SG" when="2019-10-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="area">
+                    <placeName xml:lang="geez" xml:id="n1">አባዛዢ፡</placeName>
+                    <placeName xml:lang="geez" corresp="#n1">ʾAbāzāži</placeName>
```
Comment (thea-m) [inline new/LOC7318Abazaz.xml]: please correct "gez" here and in the other files

File: new/LOC7318Abazaz.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7318Abazaz" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾAbāzāž</title>
```
Comment (thea-m) [inline new/LOC7318Abazaz.xml]: the name given in title is different than in `<placeName>`, amybe they are alternative names?

File: new/LOC7321Kanawari.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <revisionDesc>
+            <change who="SG" when="2019-10-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="area">
+                    <placeName xml:lang="geez" xml:id="n1">ከርነዋሪ፡</placeName>
+                    <placeName xml:lang="geez"  corresp="#n1">Kanawāri</placeName>
```
Comment (thea-m) [inline new/LOC7321Kanawari.xml]: fidal and transliteration don't correspond

File: new/LOC7323YadEsanat.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        <revisionDesc>
+            <change who="SG" when="2019-10-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="area">
+                    <placeName xml:lang="geez" xml:id="n1">ያፄ፡ እፃናት፡</placeName>
+                    <placeName xml:lang="geez" corresp="#n1">Yaḍe ʾƎṣānāt</placeName>
```
Comment (thea-m) [inline new/LOC7323YadEsanat.xml]: Yāḍe ʾƎḍānāt

File: new/LOC7327Qalamung.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        <revisionDesc>
+            <change who="SG" when="2019-11-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="island">
+                    <placeName xml:lang="geez" xml:id="n1">ቀለሙጅ፡</placeName>
+                    <placeName xml:lang="geez" corresp="#n1">Qalamunǧ</placeName>
```
Comment (thea-m) [inline new/LOC7327Qalamung.xml]: fidal and transliteration don't correspond

File: new/LOC7329AmadBar.xml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        <revisionDesc>
+            <change who="SG" when="2019-11-12">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="area">
+                    <placeName xml:lang="geez" xml:id="n1">ዓመድ፡  በር፡</placeName>
+                    <placeName xml:lang="geez" corresp="#n1">ʾAmad Bar</placeName>
```
Comment (thea-m) [inline new/LOC7329AmadBar.xml]: ʿĀ

File: new/LOC7330Qetama.xml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    <text>
+        <body>
+            <listPlace>
+                <place type="area">
+                    <placeName xml:lang="geez" xml:id="n1">ቄጠማ፡</placeName>
+                    <placeName xml:lang="geez" corresp="#n1">Qeṭamā</placeName>
+                </place>
+            </listPlace><!---->
+            <listRelation>
+                <relation name="gn:locatedIn" active="LOC7329AmadBar" passive="LOC1713Bagemd"/>
```
Comment (thea-m) [inline new/LOC7330Qetama.xml]: wrong ID in active

File: new/LOC7318Abazaz.xml (hunk 2 above)
Comment (sologebre) [reply new/LOC7318Abazaz.xml]: yes, it is ʾAbāzāži

File: new/LOC7321Kanawari.xml (hunk 3 above)
Comment (sologebre) [reply new/LOC7321Kanawari.xml]: Karnawāri

**Pull request merged**